rest/route/project.go

```diff
@@ -748,3 +750,76 @@ func (h *getProjectVersionsHandler) Run(ctx context.Context) gimlet.Responder {
 
 	return resp
 }
+
+type testProjectAliasHandler struct {
+	sc data.Connector
+
+	version             string
+	alias               string
+	includeDependencies bool
+	project             *dbModel.Project
+}
+
+func makeTestProjectAliasHandler(sc data.Connector) gimlet.RouteHandler {
+	return &testProjectAliasHandler{
+		sc: sc,
+	}
+}
+
+func (h *testProjectAliasHandler) Factory() gimlet.RouteHandler {
+	return &testProjectAliasHandler{
+		sc: h.sc,
+	}
+}
+
+func (h *testProjectAliasHandler) Parse(ctx context.Context, r *http.Request) error {
+	params := r.URL.Query()
+
+	h.version = params.Get("version")
+	if h.version == "" {
+		return errors.New("'version' parameter must be specified")
+	}
+	h.alias = params.Get("alias")
+	if h.alias == "" {
+		return errors.New("'alias' parameter must be specified")
+	}
+	h.includeDependencies = (params.Get("include_deps") != "")
+
+	v, err := dbModel.VersionFindOneId(h.version)
+	if err != nil {
+		return errors.Errorf("version %s not found", h.version)
```

Comment:
Nit: '%s'

Reply:
Also the error is being suppressed in this line, I'm not sure if that's what you intended.

Reply:
Deleted because it's handled elsewhere now